scripts/lib/mkcloud-libvirt.sh

```diff
@@ -320,23 +320,46 @@ function libvirt_do_onhost_deploy_image()
     fi
 }
 
+# create the libvirt configuration of a node (compute, controller, storage, lonely)
+function libvirt_do_onhost_create_compute_config()
```

Comment:
imho this should not be called _do_, as that indicates that a corresponding implementation in dirmaint backend is needed. I would name it _libvirt_create_compute_config.. or similar.